test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt

```diff
@@ -45,6 +48,24 @@ object ArgsHelper {
         }
     }
 
+    fun evaluateFilePath(filePath: String): String {
+        var file = filePath.trim().replaceFirst("~", System.getProperty("user.home"))
```

Comment:
~ should be replaced only if it's the first character in the string
https://stackoverflow.com/a/7163455

the regex should be `^~`, we should add a JUnit test for this (for both the happy path and failure case).

`~/tmp` should be replaced
`/tmp/~` should not be replaced